test/fixtures/wallet_payment_fixtures.js

```diff
@@ -58,6 +58,9 @@ const fixtures = {
         }
       }
     }
+    if (ops.email) {
```

Comment:
Is it still intended that the email defaults to `name@email.fyi` as per line 48 (I think the intention is that no email is sent if not sent in the opts)?
```
payment_info: {
        last_digits_card_number: ops.lastDigitsCardNumber || successfulLastDigitsCardNumber,
        brand: 'master-card',
        cardholder_name: 'Some Name',
        email: 'name@email.fyi'
      },
```

Reply:
No, you’re right, I missed that!

Reply:
PR to fix that: https://github.com/alphagov/pay-frontend/pull/1655